docs/device-management/5-ota/5-migrating-from-releases/README.md

```diff
@@ -0,0 +1,139 @@
+# Migrating From Releases
```

Comment:
@trond-snekvik in this document Releases, Cohorts, Deployments, etc. are all capitalized, but in the others they are (mostly) lowercase. It would be good to be consistent 👍🏻

Reply:
Good point! Turning all instances to lower case, as that seem to match the majority of instances of other concepts, like blueprints.